src/api/v4/dataview/histogram/parse-histogram-data.js

```diff
@@ -28,7 +32,7 @@ function parseHistogramData (data, nulls, totalAmount) {
    * @api
    */
   return {
-    result: _createResult(data, maxFreq),
+    result: _createResult(compactData, maxFreq),
```

Comment:
Rename to `bins`

Reply:
Better if we rename the dataview parameter to `numberOfBins`

Reply:
Forget about it. That should change in `histogram` dataview, not here.